examples/with-redis/pages/api/addemail.js

```diff
@@ -1,37 +1,31 @@
-import redis from 'redis'
-import { promisify } from 'util'
+import Redis from 'ioredis'
+import {fixUrl} from "./utils";
 
-export default async function addEmail(req, res) {
-  const client = redis.createClient({
-    url: process.env.REDIS_URL,
-  })
-  if (process.env.REDIS_PASSWORD) {
-    client.auth(process.env.REDIS_PASSWORD)
-  }
-  const saddAsync = promisify(client.sadd).bind(client)
+module.exports = async(req, res) => {
+    let redis = new Redis(fixUrl(process.env.REDIS_URL));
 
-  const body = req.body
-  const email = body['email']
+    const body = req.body;
+    const email = body["email"];
 
-  client.on('error', function (err) {
-    throw err
-  })
+    redis.on("error", function(err) {
+        throw err;
+    });
 
-  if (email && validateEmail(email)) {
-    await saddAsync('emails', email)
-    client.quit()
-    res.json({
-      body: 'success',
-    })
-  } else {
-    client.quit()
-    res.json({
-      error: 'Invalid email',
-    })
-  }
+    if (email && validateEmail(email) ) {
```

Comment:
Could we use the same Prettier settings as before? Maybe add to the package.json?